docs/using/selecting.md

```diff
@@ -195,7 +195,7 @@ It contains:
 `jupyter/pytorch-notebook` includes popular Python deep learning libraries.
 
 - Everything in `jupyter/scipy-notebook` and its ancestor images
-- [pytorch](https://pytorch.org/) machine learning library
+- [pytorch](https://pytorch.org/) machine learning library (optionally with CUDA)
```

Comment:
I think we should add a note section, which says that you should add a `cuda-` or `cuda11-` prefix to the tag to get GPU-enabled versions and explain the difference.

Reply:
Added:

> **GPU Acceleration:** Append a CUDA version prefix (`cuda12-` for CUDA 12 or `cuda11-` for CUDA 11) to the image tag to allow PyTorch operations to use compatible NVIDIA GPUs for accelerated computation.